blocks/video-player-block/features/video-player/default.jsx

```diff
@@ -18,10 +18,12 @@ const DescriptionText = styled.p`
 
 const AlertBadge = styled.span`
   background-color: #db0a07;
-  border-radius: 5px;
+  border-radius: 24px;
```

Comment:
There is a JS version of calculateRem in the CSS Framework:  https://github.com/WPMedia/news-theme-css/blob/488552a606713f52a14fc37ba8df8f7c01c12fe9/js/framework.js#L3. I prefer we either directly use Rems instead of pixels or use calculateRem.  The calculateRem is nice for devs who are not as CSS savy and then can see the px values that will be translated

Reply:
updated!